web/src/site-admin/SiteAdminLsifJobPage.tsx

```diff
@@ -0,0 +1,186 @@
+                                <td>
+                                    {jobOrError.processedOn ? (
+                                        <Timestamp date={jobOrError.processedOn} noAbout={true} />
+                                    ) : (
+                                        <span className="text-muted">Job has not yet started.</span>
+                                    )}
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <td>
+                                    {jobOrError.state === GQL.LSIFJobState.ERRORED && jobOrError.finishedOn
+                                        ? 'Failed'
+                                        : 'Finished'}{' '}
+                                    processing
+                                </td>
+                                <td>
+                                    {jobOrError.finishedOn ? (
+                                        <Timestamp date={jobOrError.finishedOn} noAbout={true} />
+                                    ) : (
+                                        <span className="text-muted">Job has not yet completed.</span>
+                                    )}
+                                </td>
+                            </tr>
+
+                            {Object.keys(jobOrError.args || {}).length > 0 && (
+                                <tr>
+                                    <td>Args</td>
+                                    <td className="job-arguments-container">
+                                        <JobArgumentsComponent args={jobOrError.args} />
+                                    </td>
+                                </tr>
+                            )}
+
+                            {(jobOrError.stacktrace || []).length > 0 && (
+                                <tr>
+                                    <td>Stacktrace</td>
+                                    <td className="job-stacktraces-container">
+                                        <JobStacktracesComponent stacktraces={jobOrError.stacktrace || []} />
+                                    </td>
+                                </tr>
+                            )}
+                        </tbody>
+                    </table>
+                </div>
+            )}
+        </div>
+    )
+}
+
+/**
+ * Construct a meaningful description from the job name and args.
+ *
+ * @param job The job instance.
+ */
+export function lsifJobDescription(job: GQL.ILSIFJob): string {
+    if (job.name === 'convert') {
+        const { repository, commit, root } = job.args as {
+            repository: string
+            commit: string
+            root: string
+        }
+
+        return `Convert upload for ${repository} at ${commit.substring(0, 7)}${root !== '' &&
+            root !== '/' &&
+            `, ${root}`}`
+    }
+
+    const internalJobs: { [K: string]: string } = {
```

Comment:
Uppercase identifiers are used only for types (in a mapped type `K` is a type, but this is just a plain index signature)

```suggestion
    const internalJobs: { [name: string]: string } = {
```

Reply:
@efritz nit: but this comment is resolved but the change wasn't amde

Reply:
Oops. Pushed.